install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -359,6 +359,9 @@ data:
   enable-external-ips: {{ .Values.global.externalIPs.enabled | quote }}
 {{- end }}
 {{- end }}
+{{- if eq .Values.global.kubeProxyReplacement "disabled" }}
```

Comment:
This looks like it should be negated as well.

Reply:
Agreed this should be negated and probably we should handle it only when kubeproxyreplacement is enabled.

Reply:
Done